src/test/kotlin/app/cash/licensee/SpdxLicensesTest.kt

```diff
@@ -18,131 +18,35 @@ package app.cash.licensee
-      |    "seeAlso": [
-      |      "http://example.com/foo"
-      |    ]
-      |  },
-      |  {
-      |    "licenseId": "BAR-1.0",
-      |    "name": "Bar License",
-      |    "reference": "https://spdx.org/licenses/BAR-1.0.html",
-      |    "seeAlso": [
-      |      "https://example.com/bar"
-      |    ]
-      |  }
-      |]}
-      |
-    """.trimMargin()
-    val spdxLicenses = SpdxLicenses.parseJson(json)
-    assertThat(
-      spdxLicenses.findByUrl("http://example.com/foo"),
-    ).isNotNull().containsExactly(
-      SpdxLicense("FOO-1.0", "Foo License", "https://example.com/foo"),
-    )
-    assertThat(
-      spdxLicenses.findByUrl("https://example.com/foo"),
-    ).isNotNull().containsExactly(
-      SpdxLicense("FOO-1.0", "Foo License", "https://example.com/foo"),
-    )
-    assertThat(spdxLicenses.findByUrl("http://example.com/bar")).isNull()
-    assertThat(
-      spdxLicenses.findByUrl("https://example.com/bar"),
-    ).isNotNull().containsExactly(
-      SpdxLicense("BAR-1.0", "Bar License", "https://example.com/bar"),
-    )
-  }
-
-  @Test fun spdxUrlSupported() {
-    val json = """
-      |{"licenses":[
-      |  {
-      |    "licenseId": "FOO-1.0",
-      |    "name": "Foo License",
-      |    "reference": "https://spdx.org/licenses/FOO-1.0.html",
-      |    "seeAlso": [
-      |      "http://example.com/foo"
-      |    ]
-      |  }
-      |]}
-      |
-    """.trimMargin()
-    val spdxLicenses = SpdxLicenses.parseJson(json)
-    assertThat(
-      spdxLicenses.findByUrl("http://example.com/foo"),
-    ).isNotNull().containsExactly(
-      SpdxLicense("FOO-1.0", "Foo License", "https://example.com/foo"),
-    )
-    assertThat(
-      spdxLicenses.findByUrl("https://spdx.org/licenses/FOO-1.0.html"),
-    ).isNotNull().containsExactly(
-      SpdxLicense("FOO-1.0", "Foo License", "https://example.com/foo"),
-    )
-  }
-
-  @Test fun spdxUrlFallback() {
-    val json = """
-      |{"licenses":[
-      |  {
-      |    "licenseId": "FOO-1.0",
-      |    "name": "Foo License",
-      |    "reference": "https://spdx.org/licenses/FOO-1.0.html",
-      |    "seeAlso": []
-      |  },
-      |  {
-      |    "licenseId": "BAR-1.0",
-      |    "name": "Bar License",
-      |    "reference": "https://spdx.org/licenses/BAR-1.0.html",
-      |    "seeAlso": [
-      |      "https://example.com/bar"
-      |    ]
-      |  }
-      |]}
-      |
-    """.trimMargin()
-    val spdxLicenses = SpdxLicenses.parseJson(json)
-    assertThat(
-      spdxLicenses.findByIdentifier("FOO-1.0"),
-    ).isEqualTo(
-      SpdxLicense("FOO-1.0", "Foo License", "https://spdx.org/licenses/FOO-1.0.html"),
-    )
-    assertThat(
-      spdxLicenses.findByIdentifier("BAR-1.0"),
-    ).isEqualTo(
-      SpdxLicense("BAR-1.0", "Bar License", "https://example.com/bar"),
-    )
+  @Test fun spdxIdsAreValidGroovy() {
+    val file = File(System.getProperty("generatedSpdxFile")).readText()
+    Assert.assertTrue('`' !in file)
```

Comment:
```suggestion
    assertThat(file).doesNotContain("`")
```
will need import changes, too